File: JAVA/Program/Orang.java
import java.util.ArrayList;
public class Orang{
    private String nama;
    private int usia;
    private String jenisKelamin;

    //constructor
    public Orang(){
    }   

    public Orang(String nama, int usia, String jenisKelamin){
        this.nama = nama;
        this.usia = usia;
        this.jenisKelamin = jenisKelamin;
    }

    //getter dan setter
    public String getNama(){
        return nama;
    }
    public void setNama(String nama){
        this.nama = nama;
    }
    public int getUsia(){
        return usia;
    }
    public void setUsia(int usia){
        this.usia = usia;
    }
    public String getJenis(){
        return jenisKelamin;
    }
    public void setJenis(String jenisKelamin){
        this.jenisKelamin = jenisKelamin;
    }
}

File: PYTHON/Program/Orang.py
class Orang:
    def __init__(self, nama ="", usia=0, jenis_kelamin =""):
        self.nama = nama
        self.usia = usia
        self.jenis_kelamin = jenis_kelamin


 #getter dan setter
    def getNama(self):
        return self.nama

    def setNama(self, nama):
        self.nama = nama

    def getUsia(self):
        return self.usia

    def setUsia(self, usia):
        self.usia = usia

    def getJenis(self):
        return self.jenis_kelamin

    def setJenis(self, jenis_kelamin):
        self.jenis_kelamin = jenis_kelamin

